indico/modules/events/registration/fields/choices.py

```diff
@@ -77,18 +79,42 @@ def _stringify_uuid(self, data, **kwargs):
         return data
 
 
+class ChoiceSetupSchema(mm.Schema):
+    with_extra_slots = fields.Bool(load_default=False)
+    choices = fields.List(fields.Nested(ChoiceItemSchema), required=True, validate=not_empty)
+
+    @pre_load
+    def _generate_new_uuids(self, data, **kwrags):
```

Comment:
Couldn't we move this `pre_load` into `ChoiceItemSchema` instead?

Reply:
no, we need to have access to the mapping here for the default item